src/SwedbankPay.Sdk/PaymentOrders/PaymentOrder.cs

```diff
@@ -0,0 +1,130 @@
+                                httpOperation.Request, client, m => new CouldNotPostTransactionException(httpOperation.Href, m));
+                        break;
+                    case PaymentOrderResourceOperations.UpdatePaymentOrderUpdateOrder:
+                        operations.Update = new ExecuteRequestWrapper<PaymentOrderUpdateRequestContainer, PaymentOrderResponseContainer>(
+                            httpOperation.Request, client, m => new CouldNotUpdatePaymentOrderException(httpOperation.Href, m));
+                        break;
+                    case PaymentOrderResourceOperations.UpdatePaymentOrderAbort:
+                        operations.Abort = new ExecuteWrapper<PaymentOrderResponseContainer>(
+                            httpOperation.Request, client, m => new CouldNotPostTransactionException(httpOperation.Href, m),
+                            new PaymentAbortRequestContainer());
+                        break;
+                    case "view-paymentorder":
+                        operations.View = httpOperation;
+                        break;
+                }
+
+                Operations = operations;
+            }
+
+            SwedbankPayOptions = swedbankPayOptions;
+        }
+
+
+        public Operations Operations { get; }
+        public PaymentOrderResponse PaymentOrderResponse { get; }
+        internal SwedbankPayOptions SwedbankPayOptions { get; }
+
+
+        internal static async Task<PaymentOrder> Create(PaymentOrderRequest paymentOrderRequest,
+                                                        SwedbankPayHttpClient client,
+                                                        SwedbankPayOptions swedbankPayOptions,
+                                                        PaymentOrderExpand paymentOrderExpand = PaymentOrderExpand.None)
+        {
+            var url = $"/psp/paymentorders{GetExpandQueryString(paymentOrderExpand)}";
+            paymentOrderRequest.SetRequiredMerchantInfo(swedbankPayOptions);
+
+            var payload = new PaymentOrderRequestContainer(paymentOrderRequest);
+
+            Exception OnError(ProblemsContainer m)
+            {
+                return new CouldNotPlacePaymentOrderException(payload, m);
+            }
+
+            var paymentOrderResponseContainer =
+                await client.HttpRequest<PaymentOrderResponseContainer>(HttpMethod.Post, url, OnError, payload);
+
+            return new PaymentOrder(paymentOrderResponseContainer, client, swedbankPayOptions);
+        }
+
+
+        internal static async Task<PaymentOrder> Get(string id,
+                                                     SwedbankPayHttpClient client,
+                                                     SwedbankPayOptions swedbankPayOptions,
+                                                     PaymentOrderExpand paymentOrderExpand = PaymentOrderExpand.None)
+        {
+            if (string.IsNullOrEmpty(id))
+                throw new CouldNotFindPaymentException(id);
```

Comment:
Isn't this quite misleading? It's an input check error, the payment order might still exist?